remove logL cut and irrelevant comments

galaxyumi opened the pull request:
logL = -9.999 are now dealt when generating the spec grid. Thus, there is no need to make logL cut when generating input ASTs. See #486 and #491

karllark commented:
Would it be worth putting in a check for such cases here and then raising a warning/error if it happens?  Just wondering if this issue might happen again and silently sneak through.

galaxyumi commented:
> Would it be worth putting in a check for such cases here and then raising a warning/error if it happens? Just wondering if this issue might happen again and silently sneak through.

It might be safer. I added a warning raiser, and added back two lines to cut out logL=-9.999 SED models just in case.

karllark commented on beast/observationmodel/ast/make_ast_input_list.py:
```diff
@@ -133,15 +133,16 @@ def pick_models_toothpick_style(
-    # Note that i speak of fluxes, but I've recently modified this to
-    # work with mags instead
+    # Check if logL=-9.999 model points sliently sneak through
+    if min(modelsedgrid.grid["logL") < -9:
```
There is a missing `]` in this line right before the closing `)`

karllark commented:
Thanks for adding in this warning and the cut.  Great idea.  Syntax error needs fixing and then this will be ready for merging.
